vip-core/src/main/java/fr/insalyon/creatis/vip/core/server/business/SpringConfigServer.java

```diff
@@ -244,6 +244,11 @@ public int getMyProxyMinHours() {
         return env.getRequiredProperty(CoreConstants.LAB_MYPROXY_MIN_HOURS, Integer.class);
     }
 
+	@Override
+	public boolean getMyProxyEnabled() {
+		return env.getRequiredProperty(CoreConstants.LAB_MYPROXY_ENABLED, Boolean.class);
```

Comment:
I had in mind to make it optional (but true by default).
And also to make the other `myproxy.*` config required only if `myproxy.enabled ` is true (or absent)